package.json

```
@@ -3,11 +3,13 @@
   "name": "gitpod-web-extension",
   "version": "0.0.1",
   "license": "UNLICENSED",
-  "description": "Browser extension (Chrome/Firefox) for Enhancing Github and Gitlab with Gitpod links",
+  "description": "Browser extension (Chrome/Firefox) for enhancing GitHub, GitLab and BitBucket with Gitpod links",
```

Official casing seems to be `Bitbucket`: https://bitbucket.org/ (lowercase second "b")

```suggestion
  "description": "Browser extension (Chrome/Firefox) for enhancing GitHub, GitLab and Bitbucket with Gitpod links",
```

---

Indeed. My bad. Thanks for the hint.